docs/src/pages/index.mdx

```diff
@@ -13,6 +13,10 @@ soon!
 
 ```bash copy
 npm install uploadthing @uploadthing/react react-dropzone
+
+# or
```

Comment:
Would like these to be broken off, ideally we'd keep all the solid stuff in it's own page

Vast majority of users won't be using solid so we should keep that docs page mostly isolated if that makes sense?